src/react/views/admin/index.js

```diff
@@ -1,10 +1,15 @@
 
 import React from 'react';
 import AdminLayout from '../../templates/admin/admin-layout';
-
+import RegularForm from '../../views/regularform'
```

Comment:
Did you add this file? It doesn't seem to be part of the commit. Is this a view, or should it be in the components subdirectory?